events.go

```diff
@@ -312,8 +314,98 @@ type PipelineEvent struct {
+		Namespace         string      `json:"namespace"`
+		VisibilityLevel   int         `json:"visibility_level"`
+		PathWithNamespace string      `json:"path_with_namespace"`
+		DefaultBranch     string      `json:"default_branch"`
+	} `json:"project"`
+	Commit struct {
+		ID        string    `json:"id"`
+		Message   string    `json:"message"`
+		Timestamp time.Time `json:"timestamp"`
+		URL       string    `json:"url"`
+		Author    struct {
+			Name  string `json:"name"`
+			Email string `json:"email"`
+		} `json:"author"`
+	} `json:"commit"`
+	Builds []struct {
+		ID         int    `json:"id"`
+		Stage      string `json:"stage"`
+		Name       string `json:"name"`
+		Status     string `json:"status"`
+		CreatedAt  string `json:"created_at"`
+		StartedAt  string `json:"started_at"`
+		FinishedAt string `json:"finished_at"`
+		When       string `json:"when"`
+		Manual     bool   `json:"manual"`
+		User       struct {
+			Name      string `json:"name"`
+			Username  string `json:"username"`
+			AvatarURL string `json:"avatar_url"`
+		} `json:"user"`
+		Runner        string `json:"runner"`
+		ArtifactsFile struct {
+			Filename string `json:"filename"`
+			Size     string `json:"size"`
+		} `json:"artifacts_file"`
+	} `json:"builds"`
+}
+
+//BuildEvent represents a build event
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ce/web_hooks/web_hooks.html#build-events
+type BuildEvent struct {
+	ObjectKind        string `json:"object_kind"`
+	Ref               string `json:"ref"`
+	Tag               bool   `json:"tag"`
+	BeforeSha         string `json:"before_sha"`
+	Sha               string `json:"sha"`
+	BuildID           int    `json:"build_id"`
+	BuildName         string `json:"build_name"`
+	BuildStage        string `json:"build_stage"`
+	BuildStatus       string `json:"build_status"`
+	BuildStartedAt    string `json:"build_started_at"`
+	BuildFinishedAt   string `json:"build_finished_at"`
+	BuildDuration     string `json:"build_duration"`
+	BuildAllowFailure bool   `json:"build_allow_failure"`
+	ProjectID         int    `json:"project_id"`
+	ProjectName       string `json:"project_name"`
+	User              struct {
+		ID    int    `json:"id"`
+		Name  string `json:"name"`
+		Email string `json:"email"`
+	} `json:"user"`
+	Commit struct {
+		ID          int    `json:"id"`
+		Sha         string `json:"sha"`
+		Message     string `json:"message"`
+		AuthorName  string `json:"author_name"`
+		AuthorEmail string `json:"author_email"`
+		Status      string `json:"status"`
+		Duration    string `json:"duration"`
+		StartedAt   string `json:"started_at"`
+		FinishedAt  string `json:"finished_at"`
+	} `json:"commit"`
+	Repository struct {
+		Name            string `json:"name"`
+		GitSSHURL       string `json:"git_ssh_url"`
+		Description     string `json:"description"`
+		Homepage        string `json:"homepage"`
+		GitHTTPURL      string `json:"git_http_url"`
+		VisibilityLevel int    `json:"visibility_level"`
+	} `json:"repository"`
```

Comment:
Same goes for this event. Please use the existing structs, unless there is a difference that requires a separate struct to be defined.